docs/resources/clickhouse.md

```diff
@@ -142,8 +142,8 @@ Optional:
 
 Required:
 
-- `integration_type` (String) Type of the service integration. Supported integrations are `clickhouse_kafka` and `clickhouse_postgresql`.
```

Comment:
We have tests for these two as a regular service integrations. Not sure if they should be here.